Source/cmakemain.cxx

```diff
@@ -73,6 +73,8 @@ static const char* cmDocumentationOptions[][2] = {
   { "--debug-output", "Put cmake in a debug mode." },
   { "--trace", "Put cmake in trace mode." },
   { "--trace-expand", "Put cmake in trace mode with variable expansion." },
+  { "--trace-file=file", "Trace only this CMake file/module. Multiple options "
```

Comment:
Other argument documentation here uses `<file>` to indicate that it isn't literally `file` (and angle brackets to indicate that it is required).